src/components/CompareResults/beta/RunValues.tsx

```diff
@@ -23,12 +24,26 @@ const styles = {
 };
 
 function RunValues(props: RunValuesProps) {
-  const { name, median, values, stddev, stddevPercent } = props.revisionRuns;
+  const {
+    name,
+    median,
+    values,
+    application,
+    stddev,
+    stddevPercent,
+    measurementUnit,
+  } = props.revisionRuns;
   return (
     <div className={styles.container}>
-      <div>
-        <b>{name}:</b> {median} ms
-      </div>
+      {application ? (
+        <div>
+          <b>{name}:</b> {median} {measurementUnit} ({application})
```

Comment:
Is there a way to show this with the fake data?

Reply:
There is an example with the fakedata, look for B: 4 N:5.

Reply:
I don't see it for this specific one, am I missing something?

![image](https://github.com/mozilla/perfcompare/assets/454175/c35d62c6-8200-45c6-82bd-e772bf303b9a)

However I do see it for others that also have B:4 N:5.

Can you please check that it's normal that `cpstartup content-process-startup 9d5066525489` doesn't show it?

Reply:
For '[cross_origin_pageload](https://firefox-source-docs.mozilla.org/testing/perfdocs/talos.html#cross-origin-pageload) [9d5066525489](https://treeherder.mozilla.org/jobs?repo=mozilla-central&revision=9d50665254899d8431813bdc04178e6006ce6d59)' you can see the application, but for '[cpstartup](https://firefox-source-docs.mozilla.org/testing/perfdocs/talos.html#cpstartup) content-process-startup [9d5066525489](https://treeherder.mozilla.org/jobs?repo=mozilla-central&revision=9d50665254899d8431813bdc04178e6006ce6d59)' there's no application available.